app/api/margatsni/v1/endpoints/users.rb

```diff
@@ -36,6 +40,26 @@ def represent_user_with_token(user)
 
             represent_user_with_token(user)
           end
+
+          namespace :me do
+            desc 'profile'
+            get do
+              represent_current_user(current_user)
+            end
+
+            desc 'edit user'
+            params do
+              requires :username, type: String, allow_blank: false
+              requires :email, type: String, regexp: User::EMAIL_REGEXP
+              requires :password, type: String, allow_blank: false
+              requires :bio, type: String, allow_blank: true
+            end
+            put do
+              current_user.update(declared(params))
```

Comment:
add `include_missing: false`

Reply:
We have a 'bio' parameter, and it does not necessarily have value. For other parameters, we have a validator 'allow_blank: false'